packages/grafana-data/src/transformations/transformDataFrame.ts

```diff
@@ -1,37 +1,71 @@
+import { Observable, of } from 'rxjs';
+import { map, mergeMap } from 'rxjs/operators';
+
 import { DataFrame, DataTransformerConfig } from '../types';
 import { standardTransformersRegistry } from './standardTransformersRegistry';
 
+const getTransform = (config: DataTransformerConfig, data: Observable<DataFrame[]>): Observable<DataFrame[]> => {
+  const info = standardTransformersRegistry.get(config.id);
+
+  if (!info) {
+    return data;
+  }
+
+  return data.pipe(
+    mergeMap(before => {
+      const defaultOptions = info.transformation.defaultOptions ?? {};
```

Comment:
This could probably be its own hoc-operator that takes in a configuration and an operator and returns an operator.

Reply:
Not sure I follow, could you please evolve that?